src/mount.ts

```diff
@@ -29,6 +31,7 @@ interface MountingOptions {
     plugins?: Plugin[]
     mixins?: ComponentOptions[]
     mocks?: Record<string, any>
```

Comment:
Typing Vue components is very difficult, will need to investigate the Vue codebase to figure out how best to do this

Reply:
I noticed this too. Am interested to hear about what you find.